README.md

```diff
@@ -50,21 +50,24 @@ for the most common UNIX, Linux, macOS, SunOS, Android and Windows command-line
 
 ## How do I use it?
 
+> [!TIP]
+> For browsing without installing a client to your computer,
```

Comment:
```suggestion
> For browsing without installing a client on your computer,
```
Although *to* is grammatically correct, *on* is more commonly used in this context